File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
import java.util.*;
import java.io.*;

class Solution {
    public boolean solution(String s) throws IOException {
        boolean answer = true;
        int length = s.length();
        
        if (length != 4 && length != 6) return false;
        
        for (char c: s.toCharArray()) {
            int numeric = Character.getNumericValue(c);
            numeric += 48;
            
            if (numeric < 48 || numeric > 57) {
                answer = false;
                break;
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    int n = s.size();
    
    if (n != 4 && n != 6) return false;
    
    
    for (int i = 0; i < n; i++) {
        if (!(s[i] >= '0' && s[i] <= '9')) {
            return false;
        }
    }
    
    return true;
}